include/RAJA/exec-cuda/forallN_cuda.hxx

```diff
@@ -112,6 +112,18 @@ struct CudaDim {
   }
 };
 
+RAJA_INLINE
```

Comment:
It's generalize to allow mappings to different cuda dimensions.... We don't have that flexibility for the 1d forall cuda_exec policy.

We COULD use the forallN execution policy for forall... that would help clean things up a bit.